pgjdbc/src/test/java/org/postgresql/jdbc/InsertRewriteWithAlternatingTypesIssue584.java

```diff
@@ -59,7 +59,7 @@ public void tearDown() throws SQLException {
   }
 
   @Override
-  protected void updateProperties(Properties props) {
+  protected void updateProperties(PGProperties props) {
```

Comment:
I'm not really sure. This is more of a POC to see if it works. Once we get it working we can fix that